### PR TITLE
Running the tests broke local demos.

### PR DESCRIPTION
**Pull request by cramforce**

### CODE REVIEW

File: build-system/internal-version.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ exports.TOKEN = getToken();
 
 function getToken() {
   var task = process.argv[2];
-  if (!task || task == 'build') {
+  if (!task || task == 'build' || task == 'test') {
```
Comment (erwinmombay) [inline build-system/internal-version.js]: no need for dist right?

Comment (cramforce) [inline build-system/internal-version.js]: This is a white list to make sure we don't go in here by accident. `dist` wants the production path.

**Comment by cramforce** [comment]
Added comments

**Comment by cramforce** [comment]
Merging trivial change.

**Pull request merged**